integrationTests/ts/__tests__/integration.test.ts

```diff
@@ -29,19 +38,19 @@ import {
   maxMessages,
   messageBatchDepth,
 } from "./utils/constants";
-import { Keypair, PCommand, PrivKey, PubKey } from "maci-domainobjs";
-import { homedir } from "os";
+import { ITestSuite, Subsidy } from "./utils/interfaces";
 import { expectSubsidy, expectTally, genTestUserCommands, isArm } from "./utils/utils";
-import { genPubKey, genRandomSalt } from "maci-crypto";
-import { existsSync, readFileSync, readdir, unlinkSync } from "fs";
-import { expect } from "chai";
+
+chai.use(chaiAsPromised);
+
+const { expect } = chai;
```

Comment:
can we keep it in one line and do `import chai, { expect } from "chai";` ?

Reply:
Unfortunately, no. Before it I need to promisify chai to make it work with typedoc.

Reply:
Fair enough!